service/hook/default_response_provider.go

```diff
@@ -27,13 +27,19 @@ type SuccessRespModel struct {
 // TransformResponseModel ...
 type TransformResponseModel struct {
 	Errors                  []string                             `json:"errors,omitempty"`
+	SkipResponses           []bitriseapi.SkipAPIResponseModel    `json:"skipped,omitempty"`
 	SuccessTriggerResponses []bitriseapi.TriggerAPIResponseModel `json:"success_responses"`
 	FailedTriggerResponses  []bitriseapi.TriggerAPIResponseModel `json:"failed_responses,omitempty"`
 }
 
 // TransformResponse ...
 func (hp DefaultResponseProvider) TransformResponse(input hookCommon.TransformResponseInputModel) hookCommon.TransformResponseModel {
 	httpStatusCode := 201
+
+	if len(input.SuccessTriggerResponses) == 0 && len(input.SkipResponses) > 0 {
+		httpStatusCode = 200
```

Comment:
good idea - please update the README's related section about the possible HTTP status codes the server can return